src/widgets/hierarchical-menu/hierarchical-menu.js

```diff
@@ -46,7 +46,7 @@ hierarchicalMenu({
   [ separator=' > ' ],
   [ rootPath ],
   [ showParentLevel=true ],
-  [ limit=1000 ],
+  [ limit=30 ],
```

Comment:
> It goes against the idea of having sensible defaults, which you don't have to configure

I agree on having sensible defaults but we do not know if either 10 or 30 are sensible defaults given the user POV: it might be too low (some items are missing) or too high (UX issue).